modules/OrbitControls.js

```diff
@@ -18,13 +18,13 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 	this.userDblClick = true;
 
-	this.userZoom = false;
+	this.userZoom = true;
```

Comment:
Could you please revert change in this file? I'm going to add options to extension config which allow to enable/disable these features